hack/lib/util.sh

```diff
@@ -54,7 +55,9 @@ kube::util::wait_for_url() {
   local i
   for i in $(seq 1 "${times}"); do
     local out
-    if out=$(curl --max-time "${maxtime}" -gkfs "${url}" 2>/dev/null); then
+    # $extra_args might contain multiple flags, we do want to split words here.
+    # shellcheck disable=SC2086
+    if out=$(curl --max-time "${maxtime}" -gkfs $extra_args "${url}" 2>/dev/null); then
```

Comment:
drop this change, switching away from AlwaysAllow authorizer will remove the need to pass bearer tokens to check healthz